dora/core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -2291,6 +2291,38 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.MASTER)
           .build();
+  public static final PropertyKey MASTER_SCHEDULER_RESTORE_JOB_FROM_JOURNAL =
+      booleanBuilder(Name.MASTER_SCHEDULER_RESTORE_JOB_FROM_JOURNAL)
+          .setDefaultValue(true)
+          .setDescription("If set to true, master will restore incomplete jobs from journal. "
+          + "Turn this off if you don't want the scheduler to automatically restore jobs.")
+          .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
+          .setScope(Scope.MASTER)
+          .build();
+  public static final PropertyKey MASTER_DORA_LOAD_JOB_TOTAL_FAILURE_COUNT_THRESHOLD =
+      intBuilder(Name.MASTER_DORA_LOAD_JOB_TOTAL_FAILURE_COUNT_THRESHOLD)
+          .setDefaultValue(-1)
+          .setDescription("The load job total load failure count threshold. -1 means never fail.")
```

Comment:
Any reason we set this to never fail as default? I think in production env we should fail fast if there are a lot of exceptions. And we would record every failure which cause a lot of memory pressure on the cluster

Reply:
Because sometimes the loading takes quite long. In my last experience, it took 2-3 days to load the whole data from UFS (about 300m files 0.5PB). If we fail the job in the middle if there's too many errors, the job will never succeed to load. So what i did is to let it load as much as possible and never fail.